src/hip/handlehip.cpp

```diff
@@ -503,12 +503,11 @@ std::size_t Handle::GetGlobalMemorySize() const
 
 std::size_t Handle::GetMaxComputeUnits() const
 {
+    const std::size_t num_cu = Value(MIOPEN_DEVICE_CU{});
```

Comment:
[ref]: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/307#discussion_r513645212
Do you still prefer to change  MIOPEN_DEVICE_CU to MIOPEN_DEBUG_ENFORCE_DEVICE_CU ?

Reply:
[off-topic] Yes

Reply:
Thanks! Aside from this one naming convention (are you going to change it soon to save some CI time?), other changes seem to be all agreed upon in #307 already.

Reply:
> are you going to change it soon to save some CI time?

Sorry, I do not understand how changing the name of env var could save CI time.

> are you going to change it soon...?

No. Also this needs to be agreed with @JehandadKhan at first, because it may affect Tuna.

Reply:
> > are you going to change it soon to save some CI time?
> 
> Sorry, I do not understand how changing the name of env var could save CI time.
> 
> > are you going to change it soon...?
> 
> No. Also this needs to be agreed with @JehandadKhan at first, because it may affect Tuna.

My bad, I thought pushing new changes will trigger another round of CI builds but apparently it is avoidable :)

ping @JehandadKhan  and @alexandraBara

Reply:
@atamazov is right, these env vars are heavily used in Tuna and changing them in MIOpen would break Tuna.